Dockerfile

```
@@ -0,0 +1,19 @@
+FROM golang:1.14.13
```

Does it need to be this specific?
```suggestion
FROM golang:1.14
```